src/xrpld/app/tx/detail/Escrow.cpp

```diff
@@ -448,13 +475,15 @@ EscrowFinish::doApply()
             return tecCRYPTOCONDITION_ERROR;
     }
 
-    // NOTE: Escrow payments cannot be used to fund accounts.
     AccountID const destID = (*slep)[sfDestination];
     auto const sled = ctx_.view().peek(keylet::account(destID));
-    if (!sled)
-        return tecNO_DST;
```

Comment:
Do we need to make this amendment guarded? The check is moved to preclaim and with different error code. Tx may fail now with a different error code.

Reply:
There are 2 checks were moved from doApply to preclaim, with the same codes - 
```
if (!sleEscrow)
        return tecNO_TARGET;
if (!sleDst)
        return tecNO_DST;
```
You mess them because in original code they called 
```
      auto const slep = ctx_.view().peek(k);
     auto const sled = ctx_.view().peek(keylet::account(destID));    
```